src/js/claims-status/components/appeals-v2/Issues.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CollapsiblePanel from '../../../common/components/CollapsiblePanel';
+
+const Issues = ({ issues }) => {
+  // TO-DO: think about whether it's crazy to run through the issues list 4 times
```

Comment:
Nah. Given the relatively short list of issues, I think it's a micro-optimization to make this O(n) instead of O(4n). We're not talking hundreds of thousands of iterations, so this is good like it is--even for low-powered mobile phones, it shouldn't be an issue.